cmd/cmd/apiInfo.go

```diff
@@ -38,149 +33,136 @@ var apiName string
 const showAPICmdLiteral = "api"
 const showAPICmdShortDesc = "Get information about the specified API"
 
-var showAPICmdLongDesc = "Get information about the API specified by the flag --name, -n\n"
+var showAPICmdLongDesc = "Get information about the API specified by command line argument [apiName] or list all the apis\n"
 
 var showAPICmdExamples = dedent.Dedent(`
 Example:
-  ` + utils.ProjectName + ` ` + showCmdLiteral + ` ` + showAPICmdLiteral + ` -n TestAPI
+To get details about a specific api
+  ` + utils.ProjectName + ` ` + showCmdLiteral + ` ` + showAPICmdLiteral + ` TestAPI
+
+To list all the apis
+  ` + utils.ProjectName + ` ` + showCmdLiteral + ` ` + showAPICmdLiteral + `
+`)
+
+var showAPICmdUsage = dedent.Dedent(
+`Usage:
+  ` + utils.ProjectName + ` ` + showCmdLiteral + ` ` + showAPICmdLiteral + ` [apiName]
 `)
 
 // apiShowCmd represents the show api command
 var apiShowCmd = &cobra.Command{
-	Use:   showAPICmdLiteral,
-	Short: showAPICmdShortDesc,
-	Long: showAPICmdLongDesc + showAPICmdExamples,
-	Run: func(cmd *cobra.Command, args []string) {
-		utils.Logln(utils.LogPrefixInfo+"Show API called")
-		executeGetAPICmd(apiName)
-	},
+    Use:   showAPICmdLiteral,
+    Short: showAPICmdShortDesc,
+    Long:  showAPICmdLongDesc + showAPICmdExamples,
+    Run: func(cmd *cobra.Command, args []string) {
+        utils.Logln(utils.LogPrefixInfo + "Show API called")
+        if len(args) == 0 {
+            executeListAPIsCmd()
+        }else if len(args) == 1{
```

Comment:
Don't we need to have a space after the curly braces? Let's fix it in other places as well.